api/core/request_fields_test.go

```diff
@@ -270,6 +270,46 @@ func TestBadPostFilesStoreFail(test *testing.T) {
     }
 }
 
+func TestBadPostFilesFileSizeExceeded(test *testing.T) {
+    resetMaxFileSize := config.WEB_MAX_FILE_SIZE_KB.Get()
+
+    // Set size to 1 KB for testing, then reset when done testing.
+    config.WEB_MAX_FILE_SIZE_KB.Set(1);
+    defer config.WEB_MAX_FILE_SIZE_KB.Set(resetMaxFileSize)
+
+    endpoint := `/test/api/post-files/bad/size-exceeded`;
+
+    type requestType struct {
+        APIRequestCourseUserContext
+        MinRoleStudent
+
+        Files POSTFiles
+    }
+
+    handler := func(request *requestType) (*any, *APIError) {
+        return nil, nil;
+    }
+
+    routes = append(routes, NewAPIRoute(endpoint, handler));
+
+    // Two paths provided: a.txt is under the size limit, 1092bytes.txt is over the size limit.
+    paths := []string{
```

Comment:
You can just drop a.txt from your paths in this test.
We already tested that it worked in a previous test, so we will want to keep it simple.